src/components/AppDetails/Versions.js

```diff
@@ -120,28 +125,34 @@ const VersionsTable = ({
                             {moment(version.created).format('ll')}
                         </TableCell>
                         <TableCell>
-                            <Button
-                                small
-                                secondary
-                                className={styles.installBtn}
-                                disabled={
-                                    version.version === installedVersion ||
-                                    versionBeingInstalled
-                                }
-                                onClick={() => onVersionInstall(version)}
-                            >
-                                {isVersionInstalling && (
-                                    <>
-                                        {i18n.t('Installing...')}
-                                        <CircularLoader small />
-                                    </>
-                                )}
-                                {!isVersionInstalling
-                                    ? version.version === installedVersion
-                                        ? i18n.t('Installed')
-                                        : i18n.t('Install')
-                                    : ''}
-                            </Button>
+                            <div className={styles.tableActions}>
```

Comment:
this was necessary otherwise the two buttons were not aligned properly when the loader is shown

Reply:
I think [this component](https://ui.dhis2.nu/demo/?path=/story/button-strip--default) is supposed to do the same thing. But I don't know if it works with the link

Reply:
yeah that worked - updated it. Thanks @Mohammer5